### PR TITLE
New patrol + Patrol Art

### PR DESCRIPTION
**Pull request by SnapshotManor**
_To Scribble, who is overworked (ILY):_
_This is low priority, so you can ignore it for as long as you'd like :)_

## About The Pull Request

- Adds 1 new patrol art piece with 5 variants
- Added a new variant of patrol art "fst_hunt_mice_ohnotheyrehot" called "gen_warrior_two_outsiders"
- Added 1 new patrol ID: "gen_bord_twocats_leafbare" with 8 possible outcomes
- Adds "gen_cat_dead_EXPLICIT" IDs to the explicit art json

## Why This Is Good For ClanGen

This PR adds a fresh new patrol, as well as utilizing underused patrol art, on top of adding new patrol pieces. 

## Proof of Testing

5notagentsmith

### CODE REVIEW

**Comment by SnapshotManor** [comment]
I'm not sure why the pronoun check isn't working. I redownloaded the border patrol file and pasted my additions to it, yet it's flagging things I did not change. Odd. Maybe it's just getting it wrong somehow lol

**Comment by anjubatus** [comment]
i looked at the commit that gota  failed test, i noticed one of these at least is misspelled?
kuva
i'm not sure if it will fix the issue though

File: resources/dicts/patrols/general/border.json [hunk 1]
```diff
@@ -1,2536 +1,2887 @@
...
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["dislike"],
+                        "amount": -5
+                    }
+                ],
+                "other_clan_rep": -1
+            },
+            {
+                "text": "Deciding it looks safe enough, p_l sneaks across the border... right into a late-night border patrol. {PRONOUN/p_l/subject/CAP}'{VERB/p_l/re/s} dealt with pretty quickly, and {PRONOUN/p_l/poss} fur prickles with embarrassment as {PRONOUN/p_l/subject} {VERB/p_l/scramble/scrambles} back over the border. {PRONOUN/p_l/subject} {VERB/p_l/have/has} the whole walk back to camp to think of an explanation for the fresh claw-wounds scoring {PRONOUN/p_l/poss} pelt.",
```
Comment (arisaema12) [inline resources/dicts/patrols/general/border.json]: missing /CAP after 'back over the border.'

File: resources/dicts/patrols/general/border.json [hunk 2]
```diff
@@ -1,2536 +1,2887 @@
...
+                "weight": 20
+            },
+            {
+                "text": "s_c sneaks up behind the Twoleg. With a mighty roar, s_c leaps onto its back and {PRONOUN/s_c/poss} momentum sends the Twoleg falling to the ground with a giant thump.",
+                "exp": 10,
+                "weight": 20,
+                "stat_skill": ["FIGHTER,0"]
+            },
+            {
+                "text": "s_c stops the patrol, narrowing {PRONOUN/s_c/poss} eyes. That isn't a Twoleg! {PRONOUN/s_c/subject} came across one before during {PRONOUN/s_c/object} time outside the Clan. It's a 'scarecrow', {PRONOUN/s_c/subject} {VERB/s_c/explain/explains}, meant to protect the Twoleg fields from harm. The patrol continues on their way, giving the scarecrow a wide berth.",
```
Comment (arisaema12) [inline resources/dicts/patrols/general/border.json]: Missing /CAP after 'That isn't a Twoleg!'

File: resources/dicts/patrols/general/border.json [hunk 3]
```diff
@@ -1,2536 +1,2887 @@
...
+    "season": ["any"],
+    "types": ["border"],
+    "tags": [],
+    "patrol_art": "gen_border_dogscent_clanmember",
+    "min_cats": 1,
+    "max_cats": 1,
+    "weight": 20,
+    "chance_of_success": 20,
+    "intro_text": "Having been chased by a small, yappy dog along the border, p_l is about to land a hefty blow to the irritating creature's face, when a panicked voice calls out from nearby.",
+    "decline_text": "Frankly, p_l has better things to do than talk to weird dog-defenders. {PRONOUN/p_l/subject} {VERB/p_l/head/heads} home early, without a second glance at whoever called out.",
```
Comment (arisaema12) [inline resources/dicts/patrols/general/border.json]: missing /CAP after 'weird dog-defenders'

**Review by arisaema12 — COMMENTED** [review]
I think I've found the 'Capitalization errors' causing the pronoun_test to fail, see if adding the /CAPs fixes it!

**Review by scribblecrumb — APPROVED** [review]

**Pull request merged**